batch_functions/batch_recfield.m

```diff
@@ -19,7 +19,7 @@
         'sig_alpha', 'unsmoothed_recfield_metrics', 'sig_check', 'consec_bins', ...
         'span', 'threshold_scalar', 'cluster_analysis', 'bin_gap'};
     ignore_headers = {
-        'significant', 'background_rate', 'background_std', 'threshold', 'p_val', ...
+        'significant', 'background_rate', 'background_std', 'avg_response', 'response_window_rm', 'threshold', 'p_val', ...
```

Comment:
Adding avg_response to the output of the recfield analysis should be approved by Dr. Moxon.